dask/dataframe/partitionquantiles.py

```diff
@@ -267,6 +267,11 @@ def percentiles_to_weights(qs, vals, length):
     return vals.tolist(), weights.tolist()
 
 
+def replace_nans(arr, replace_value=np.nan):
+    """Replace nans in array with provided replacement value"""
+    return [replace_value if pd.isna(x) else x for x in arr]
```

Comment:
It seems premature to add a `replace_value=` keyword and then not use it. Note this suggestion also changes the function name (`nans` --> `na`, which more accurate I think) 

```suggestion
def replace_na(arr):
    """Replace NA in array with np.nan"""
    return [np.nan if pd.isna(x) else x for x in arr]
```